authors/apps/articles/renderers.py

```diff
@@ -38,6 +38,38 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
             return json.dumps({
                 'articles': data
             })
+
```

Comment:
It's always a good practice to have two white space lines left before the definition of class. Considering that we are following the PEP 8 guideline.

Reply:
Thanks, Amos for the tip. I am going to apply pep 8 standards to the file.